merge_coda_files/merge_coda_files.py

```diff
@@ -0,0 +1,119 @@
+import argparse
+import time
+from os import path
+import json
+from dateutil.parser import isoparse
+
+from core_data_modules.cleaners import Codes
+from core_data_modules.cleaners.cleaning_utils import CleaningUtils
+from core_data_modules.traced_data import Metadata
+from core_data_modules.traced_data.io import TracedDataJsonIO, TracedDataCoda2IO
+from core_data_modules.util import IOUtils
+from core_data_modules.data_models import Scheme
+
+# opens a coding scheme
+def open_scheme(filepath):
+    with open(filepath, "r") as f:
+        firebase_map = json.load(f)
+        return Scheme.from_firebase_map(firebase_map)
+
+def get_strings_coda(key_map, code_scheme, list_td):
+    list_codeids = [code.code_id for code in code_scheme.codes]
+    for td in list_td:
+            td.append_data(
+                {new_key: code_scheme.get_code_with_id(td[old_key]["CodeID"]).string_value
+                for new_key, old_key in key_map.items()
+                if old_key in td
```

Comment:
Why do you need this test? Surely every key into the TracedDatas being used here has a label set by now?

Reply:
Now that all of your earlier stages are built, I presume this is no longer the case, and this test can be removed? Otherwise please explain each example you see of fields not containing a Label.

Reply:
Removed